doc/engineering-reference/src/simulation-models-encyclopedic-reference-003/indoor-swimming-pool.tex

```diff
@@ -152,38 +154,38 @@ \subsection{Conduction through the bottom of the pool}\label{conduction-through-
 \subsection{Makeup pool water supply}\label{makeup-pool-water-supply}
 
 \begin{equation}
-Q_{fw} = m_{fw} \cdot cw \cdot (T_p – T_{fw})
+Q_{fw} = \dot{m}_{fw} \cdot cw \cdot (T_p - T_{fw})
 \end{equation}
 
 where
 
-\(m_{fw}\) = Mass flow rate
+\(m_{fw}\) = Mass flow rate of fresh water (lb·s)
```

Comment:
Shouldn't this be lb/s ?
And why is this whole section in IP units?  I'm not going to insist that it all be changed over now, but wondering why.